tests/unit/plugins/kernel_install/utils_test.sh

```diff
@@ -533,9 +533,11 @@ function test_install_kernel_remote()
   mkdir -p "${KW_DEPLOY_TMP_FILE}/kw_pkg"
   touch "${KW_DEPLOY_TMP_FILE}/kw_pkg/kw.pkg.info"
   printf 'kernel_name=%s\n' "$name" > "${KW_DEPLOY_TMP_FILE}/kw_pkg/kw.pkg.info"
```

Comment:
As you are taking the opportunity to make this change, also add this line inside the block below.